okta/okta_util.go

```diff
@@ -0,0 +1,25 @@
+package okta
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/hex"
+
+	"github.com/pkg/errors"
+)
+
+func randomString() (string, error) {
```

Comment:
You can probably add tests to these to ensure length + not clashing after a couple tries

Reply:
Done.